src/content/blog/2025-05-05-celebrating-ten-years-of-discover.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Celebrating Ten Years of Discover
+author: Miriam Seely
+description: >-
+  We are celebrating 10 years of the Discover Basemap and Imagery Service this
+  month! Discover is our one-stop shop for high-resolution aerial imagery and
+  custom Utah basemaps, all available on our website for free and licensed use.
+date: '2025-05-05T20:08:26.922Z'
+category: Discover
+cover_image: /src/images/pillar-blog/default-social-card.png
+cover_image_alt: ugrc social card
+---
+
+import Contacts from '@components/page/Contacts.astro';
+
+We are celebrating 10 years of the Discover Basemap and Imagery Server this month! [Discover](/products/discover/) is our one-stop shop for aerial imagery and custom Utah basemap services, all available from UGRC for free to public and licensed users. Our nearly 2,000 registered accounts include government agencies such as the Utah Department of Transportation, the Utah Department of Public Safety, the Utah Division of Wildlife Resources, and the Utah Division of Forestry, Fire, and State Lands. We also serve a wide variety of local governments, private firms, K-12 and higher education, and many other organizations that use the services to benefit the lives of everyday citizens in Utah.
+
+## So, where did Discover come from?
+
+Before we implemented the current Discover service, UGRC utilized a platform called [ArcGIS Image Server]](https://enterprise.arcgis.com/en/image/latest/get-started/windows/what-is-arcgis-image-server-.htm). It was available for public and private use, but connections made to the service created a heavy digital load on the ArcGIS Server. This resulted in a number of problems for both UGRC and our user base; the services experienced rendering latenecy and occasional downtime. While this old service did provide significant customization options for imagery and topographic maps, it was not the best solution.
```

Comment:
```suggestion
Before we implemented the current Discover service, UGRC utilized a platform called [ArcGIS Image Server]](https://enterprise.arcgis.com/en/image/latest/get-started/windows/what-is-arcgis-image-server-.htm). It was available for public and private use, but connections made to the service created a heavy digital load on our ArcGIS Server. This resulted in a number of problems for both UGRC and our user base; the services experienced rendering latenecy and occasional downtime. While this old service did provide significant customization options for imagery and topographic maps, it was not the best solution.
```

Just making it clear that this was a server we had to maintain, not something Esri was providing for us.